parallel-install/pkg/jobmanager/README.md

```diff
@@ -0,0 +1,75 @@
+# `jobManager` Package
+
+This package, called `jobManager`, implements a `job` interface which is used to enable a clean automated Kyma deployment in regards of configuring the cluster and the components/resources/etc. of Kyma. The terms "Deployment" and "Deploy" are used in the context of installing Kyma on an empty cluster, or to upgrade Kyma from an older to a newer version.
+
+## Workflow / General mechanism
+
+### Requirements of this package
```

Comment:
Do you want to call it Requirements or rather call it as Features ? As this is not a proposal any more.